docs/guide/build.md

```diff
@@ -228,6 +228,7 @@ El modo librería incluye una configuración simple y pragmática para librería
 
 ## Opciones avanzadas para Base
 
+::: warning
```

Comment:
Puede que esté repetido?

Reply:
There I corrected it, I don't know why it was repeated.